libs/generators/src/lib/generators/generators.component.scss

```diff
@@ -0,0 +1,42 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  height: calc(100vh - 65px);
+  
+    .container {
+      display: grid;
+      grid-template-columns: 15rem 1fr;
+      grid-template-rows: 1fr;
+      height: 100%;
+    }
+  
+    .side-nav {
+      display: flex;
+      flex-direction: column;
+      background-color: rgb(116, 171, 238);
+    }
+  
+    .side-nav-item {
+      display: flex;
+      justify-content: center;
+      padding: 16px 8px;
+      color: white;
+      font-size: 1.25rem;
+      cursor: pointer;
+
+      &:hover {
+        background-color: rgb(123, 179, 246);
```

Comment:
Should we decide on colors and export them to a scss variables file?

Reply:
no, we should use only material colors